traitcuration/traits/datasources/zooma.py

```diff
@@ -95,7 +94,10 @@ def create_mapping_suggestion(trait, term):
     """
     Creates a mapping suggestion in the app's database, if it doesn't exist already.
     """
-    zooma = User.objects.filter(username="Zooma").first()
+    zooma = User.objects.filter(username="ZOOMA").first()
+    if zooma is None:
+        zooma = User(username="ZOOMA", email="zooma@zooma.com")
```

Comment:
Since zooma.com is a real domain (and not relevant to our ZOOMA as well), it's better to substitute the development mailing list of EVA

```suggestion
        zooma = User(username="ZOOMA", email="eva-dev@ebi.ac.uk")
```